### ISSUE TITLE
no-invalid-debug-function-arguments is not checking import source

### ISSUE THREAD

**Issue by kellyselden** [ember-cli/eslint-plugin-ember#604]
```js
import { deprecate } from './-utils';

import isHtmlSafe from './string/is-html-safe';

export default deprecate(isHtmlSafe, 'isHtmlSafe', 'string.isHtmlSafe');
```

I have my own deprecate function, but the rule is triggering on it.

**Comment by bmish** [comment]
Definitely want to fix this. Related to #590.
